src/Illuminate/Database/Query/Builder.php

```diff
@@ -948,6 +948,30 @@ protected function whereInExistingQuery($column, $query, $boolean, $not)
         return $this;
     }
 
+    /**
+     * Add a "where in raw" clause to the query.
+     *
+     * @param  string  $column
+     * @param  array   $values
+     * @param  string  $boolean
+     * @param  bool    $not
+     * @return $this
+     */
+    public function whereInRaw($column, array $values, $boolean = 'and', $not = false)
+    {
+        $type = $not ? 'NotInRaw' : 'InRaw';
+
+        if ($values instanceof Arrayable) {
```

Comment:
Because of the `array $values` typehint in the parameters definition above, `$values` is forcibly an array here.

For consistency with `whereIn()` I think the typehint should be removed.

Reply:
Thanks!